lib/react_on_rails/configuration.rb

```diff
@@ -14,19 +14,22 @@ def self.configuration
       trace: Rails.env.development?,
```

Comment:
Why are these files getting permissions changes. Sounds accidental.

```
lib/react_on_rails/configuration.rb 100644 → 100755
```

I'd change the files back to the original settings.